Source/Core/DolphinWX/Config/WiiConfigPane.cpp

```diff
@@ -39,29 +40,33 @@ void WiiConfigPane::InitializeGUI()
 	m_system_language_strings.Add(_("Traditional Chinese"));
 	m_system_language_strings.Add(_("Korean"));
 
+	m_video_mode_strings.Add(_("576i"));
+	m_video_mode_strings.Add(_("480i"));
+	m_video_mode_strings.Add(_("480p"));
```

Comment:
It would be useful to note the framerate here, because users might not know what the correlation between resolution and framerate is. `576i (50 Hz)` / `480i (60 Hz)` / `480p (60 Hz)`